geoportal/c2cgeoportal_geoportal/views/theme.py

```diff
@@ -447,12 +447,14 @@ def _fill_editable(self, layer_theme, layer):
             errors.add(str(exception))
         return errors
 
-    def _fill_child_layer(self, layer_theme, layer_name, wms, wms_layer_obj):
+    def _fill_child_layer(self, layer_theme, layer_name, wms):
+        wms_layer_obj = wms["layers"][layer_name]
         if not wms_layer_obj["children"]:
             layer_theme["childLayers"].append(wms["layers"][layer_name]["info"])
         else:
             for child_layer in wms_layer_obj["children"]:
-                self._fill_child_layer(layer_theme, layer_name, wms, wms["layers"][child_layer])
+                LOG.error(child_layer)
```

Comment:
Forgotten log ? Seems there is no real error.

Reply:
Oups, thanks :-)